workers/reporter/docker/Reporter.Dockerfile

```diff
@@ -51,4 +51,24 @@ RUN chgrp -R 0 /home/ort && chmod -R g+rwX /home/ort
 USER $USERNAME
 WORKDIR $HOMEDIR
 
+COPY --from=scancode-license-data-build --chown=$USER:$USER /opt/scancode-license-data /opt/scancode-license-data
+
 ENTRYPOINT ["/bin/bash"]
+
+# Build-Stage for Python executing scancode-license-data to get the license texts in a directory
+FROM python:3.11-slim AS scancode-license-data-build
+
+# Keep in sync with Scanner.Dockerfile
+ARG SCANCODE_VERSION=32.2.1
+
+RUN apt-get update && apt-get install -y curl libgomp1 && rm -rf /var/lib/apt/lists/*
+
+# Use pip to install ScanCode
+RUN curl -Os https://raw.githubusercontent.com/nexB/scancode-toolkit/v$SCANCODE_VERSION/requirements.txt && \
```

Comment:
I wonder whether in this case, where ScanCode itself it not really needed, it wouldn't be more efficient to use e.g. `curl` (and some `jq` magic) on the [LicenseDB API](https://github.com/aboutcode-org/scancode-licensedb?tab=readme-ov-file#api) and download the `*.LICENSE` files directly.